### PR TITLE
Add support for Fedora version 23 and 24 (building from official Dock…

### PR DESCRIPTION
**Pull request by nghiant2710**
…er Fedora base images)

### CODE REVIEW

**Comment by Page-** [comment]
SGTM

File: SHASUMS256.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 566c008a4c79233d1812bdfc6460e465e4dd13d27b333866dde60d854d06873b  fedora-minimal-armhf-21.tar.gz
 2facf0436d538253efe44b0cc1dd60cbfbcd5cb1867ff3336c76ff297da06829  fedora-minimal-armhf-22.tar.gz
 39199b7f042208ccd574f0c108c23c02e0b00bf29d64d5cb56b9b8a0e6dab98b  fedora-minimal-armhf-23.tar.gz
+fbfc143c32f5eeeb733e2be5d39a8b5ecaddf3fdaf25751001cb1b8530af2fb1  Fedora-Docker-Base-23.armhf.tar.xz
+2a82b76a1d7be0e215269f5ff1ee33b6961343da16d484da469bfcd51e9bb08d  Fedora-Docker-Base-24.armhf.tar.xz
```
Comment (petrosagg) [inline SHASUMS256.txt]: Why do we import these? We only do this for packages with very long compilation times under QEMU and for packages that a large amount of images need. I think we should be creating the rootfs in this build process

File: Dockerfile.tpl [hunk 2]
```diff
@@ -2,7 +2,48 @@ FROM #{FROM}
...
+    && dnf install -y \
+        ca-certificates \
+        tar \
+        systemd \
+        udev \
+        which \
+        hostname \
+    && dnf clean all
+
+# Install Systemd
```
Comment (petrosagg) [inline Dockerfile.tpl]: Forgotten comment?

File: Dockerfile.tpl [hunk 3]
```diff
@@ -2,7 +2,48 @@ FROM #{FROM}
...
+        systemd-remount-fs.service \
+        getty.target \
+        graphical.target \
+        console-getty.service \
+        systemd-vconsole-setup.service
+
+COPY entry.sh /usr/bin/
+COPY launch.service /etc/systemd/system/launch.service
+
+RUN systemctl enable /etc/systemd/system/launch.service systemd-udevd
```
Comment (petrosagg) [inline Dockerfile.tpl]: No need for the full path for `launch.service`

File: SHASUMS256.txt (hunk 1 above)
Comment (nghiant2710) [inline SHASUMS256.txt]: these rootfs are the build artifacts from Koji (Fedora build system), I had some issue with running Koji in a container in the past so I don't want to add it in our build process. I will look into it later then we can update the build process later

Comment (petrosagg) [inline SHASUMS256.txt]: Why do we have to use the Koji build system? Check the second example from here https://www.freedesktop.org/software/systemd/man/systemd-nspawn.html#Examples

Comment (nghiant2710) [inline SHASUMS256.txt]: I will try this way however can we save it for the next PR?

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**